public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="">
```

Comment:
```suggestion
<html lang="ja">
```

にしない理由は何かありますか？

Reply:
今回はあえて指定する理由がないという認識でした。
翻訳機能の件は別としても日本語のアプリなら `lang="ja"`は指定するべきといったことでしょうか？

Reply:
なるほどー。であれば

```suggestion
<html>
```

でいいんじゃない？と思ったりしました。（それだとvalidなHTMLじゃない、とlinterに怒られるかもしれませんが・・・）

もしかして、ネット記事か何かで`lang=""`みたいな書き方が推奨されてたんでしょうか？

> 翻訳機能の件は別としても日本語のアプリなら lang="ja"は指定するべきといったことでしょうか？

実際問題、日本語しか想定していないなら`lang="ja"`と書いてしまった方が白黒ハッキリしていいんじゃないかなーと個人的には思います〜。

Reply:
`vue create`した時からこうなっていた（手を加えていない）認識なので、Vueではこういうものなのかなぁと思っていました。
ですが、仰る通り`lang="ja"`が明確で良いと思うので、そのように修正します！

Reply:
修正しました 118dd94

Reply:
> vue createした時からこうなっていた（手を加えていない）認識なので、Vueではこういうものなのかなぁと思っていました。


なるほど、そういうことだったんですね。
個人的な推測ですが、Vue.jsとしては「lang属性は付けておかないとHTML的にvalidでないから付けておきたい、だが、どんな人が使うかわからないので具体的な言語を指定することはできない→仕方がないから`lang=""`で逃げる」という形になってるんじゃないでしょうか？
なので、利用者側が「メインで使うのはこの言語」というのを判断して`lang="ja"`のように編集してあげればいいんだと思います。